### PR TITLE
dcos node

### PR DESCRIPTION
**Pull request by mgummelt**
I was about to implement `dcos node log`, and figured we probably need a top level `dcos node` first.  It only lists slaves, because DCOS does not provide an endpoint to get the masters AFAIK.

### CODE REVIEW

**Comment by mgummelt** [comment]
@jsancio tests pass.  plz review

File: cli/dcoscli/node/main.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+def main():
+    try:
+        return _main()
+    except DCOSException as e:
+        emitter.publish(e)
+        return 1
+
+
+def _main():
```
Comment (jsancio) [inline cli/dcoscli/node/main.py]: Out of scope for this PR but should we refactor this dance (`main` and `_main`) and move it to `dcos`? I haven't thought it much but it feels useful for other sub-commands.

File: cli/tests/unit/data/node.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+ HOSTNAME       IP                          ID                   
+ dcos-01   172.17.8.101  20150630-004309-1695027628-5050-1649-S0
```
Comment (jsancio) [inline cli/tests/unit/data/node.txt]: Hmm. Would this fail if we run the tests on a developer laptop?

Comment (jsancio) [inline cli/tests/unit/data/node.txt]: I see. Ignore my comment. This is a unit test.

File: cli/dcoscli/tables.py [hunk 3]
```diff
@@ -276,3 +276,21 @@ def package_search_table(search_results):
     tb.align['DESCRIPTION'] = 'l'
 
     return tb
+
+
+def slave_table(slaves):
+    """Returns a PrettyTable representation of the provided DCOS slaves
+
+    :param slaves: slaves to render.  dicts from /mesos/state-summary
+    :type aves: [dict]
```
Comment (jsancio) [inline cli/dcoscli/tables.py]: s/aves/slaves/

**Comment by jsancio** [comment]
Looks good to me! Just some minor comments.

File: cli/dcoscli/node/main.py (hunk 1 above)
Comment (mgummelt) [inline cli/dcoscli/node/main.py]: we could, yea

**Pull request merged**